docs/TestingFabricProxyModel.rst

```diff
@@ -92,6 +94,57 @@ To run Fabric with Hyperledger Avalon, follow these steps:
 
        ./scripts/start_fabric.sh -d
 
+Setup Hyperledger Avalon Fabric Proxy Model on multiple machine
+----------------------------------------------------------------
+Multinode Hyperledger Avalon fabric proxy model setup requires certain modifications to existing
+docker files. For example to setup the fabric blockchain network on a separate machine and avalon on a separate machine
+requires below changes.
+
+1. On machine1(with ip address IP1) need to run the minifab if you are choose to setup fabric on single machine.
+   Or else if you chose the any customized way other than minifab to setup a fabric network on multiple nodes, then it is mandatory to deploy
+   the avalon chaincodes which are located in (https://github.com/hyperledger/avalon/tree/master/sdk/avalon_sdk/connector/blockchains/fabric/chaincode)
+
+   .. code:: sh
+        cd $TCF_HOME
+        ./scripts/start_fabric.sh -e
+
+   start_fabric.sh with option -e exposes the ports to host machine and it generates the network profile with host ip in network config file.
+
+2. On machine2 (with ip address IP2) where Avalon is going to setup, copy the crypto materials used to setup the fabric network to home directory.
+   Avalon shell and avalon fabric connector need these crypto materials to interact with the blockchain.
+
+   .. code:: sh
+        scp -r ~/mywork <user_name>@<IP1>:~/
+
+3. If you choose the customized setup other than the start_fabric.sh (minifab) then create the network.json as in (https://github.com/hyperledger/avalon/blob/master/sdk/avalon_sdk/connector/blockchains/fabric/network.json)
+
+4. If these 2 matchines are behind the corporate network please update environment variables `no_proxy` and `NO_PROXY`
+   with the IP1 on machine2 as mentioned above.
+
+5. Remove the network tag from the docker compose file - https://github.com/hyperledger/avalon/blob/master/docker/compose/avalon-fabric.yaml#L55
```

Comment:
Change docker to Docker here on line 124 and line 136

Reply:
Done